package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "orbit_",
   "version": "0.1.0",
-  "description": "Orbit - Communication System on IPFS",
+  "description": "Communication System on IPFS",
```

Comment:
Change this to the GitHub description: "Orbit communication protocol and library."